embed_grader/serapis/forms/assignment_forms.py

```diff
@@ -86,23 +94,45 @@ def __init__(self, *args, **kwargs):
                     initial=initial_val,
             )
         
-        num_max_team_members = assignment.num_max_team_members if assignment else 1
+        # add team_choice radio button and max_num_team_members input field
+        max_num_team_members = assignment.max_num_team_members if assignment else 1
         initial_team_choice_val, initial_num_member_val = (
-                (AssignmentForm.TEAM_OPTION_INDIVIDUAL, 2) if num_max_team_members == 1
-                else (AssignmentForm.TEAM_OPTION_TEAM, num_max_team_members))
+                (AssignmentForm.TEAM_OPTION_INDIVIDUAL, 2) if max_num_team_members == 1
+                else (AssignmentForm.TEAM_OPTION_TEAM, max_num_team_members))
 
         self.fields['team_choice'] = forms.ChoiceField(
                 required=True,
                 widget=forms.RadioSelect,
                 choices=AssignmentForm.TEAM_CHOICES,
                 initial=initial_team_choice_val,
         )
-        self.fields['num_max_team_members'] = forms.IntegerField(
+        self.fields['max_num_team_members'] = forms.IntegerField(
                 required=False,
                 initial=initial_num_member_val,
                 validators=[MinValueValidator(2)]
```

Comment:
Do we want a comma here? I think we should make it consistent, either to add the comma or remove the comma from line 107

Reply:
Good catch! Always put a comment when parameters are in several lines